modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,336 @@
+<?php declare(strict_types=1);
+/**
+ * Get Electrophysiology Session Data.
+ *
+ * This retrieves the entire electrophysiology session data
+ * for a candidate to be visible on the electrophysiology_session form.
+ *
+ * PHP Version 7
+ *
+ * @category LORIS
+ * @package  Electrophysiology_Session
+ * @author   Muhammad Khan <muhammad.khan@mcin.ca>
+ * @license  LORIS license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+require_once 'ElectrophysioFile.class.inc';
+
```

Comment:
There should be a check to see if `$_REQUEST['sessionID']` is set and is a number and return a `400 Bad Request` if it's not the case.

Reply:
ok. Done